enterprise/tasks.py

```diff
@@ -35,11 +35,13 @@ def create_enterprise_enrollment(course_id, enterprise_customer_user_id):
 
     enterprise_customer = enterprise_customer_user.enterprise_customer
     if enterprise_customer.catalog_contains_course(course_id):
+        LOGGER.info('***** TEST')
```

Comment:
Did you mean to leave this log message in?

Reply:
No will be removed soon.